### ISSUE TITLE
Season pack notification when whole season is grabbed instead of N episode notifications

### ISSUE THREAD

**Issue by rodrigogonegit** [Sonarr/Sonarr#7305]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Currently when a Season Pack is grabbed, Sonarr will send N notifications to the connected clients, one per episode. Since there is explicit support for Season Packs, this is a known entity from Sonarr POV. Imo, it would make sense to replace all those N notifications with one Pack notification.

The same could be applied for when a whole Season/bunch of Seasons are deleted. 

### Describe the solution you'd like

A Season added notification.
And ideally a Season deleted notification as well.

### Describe alternatives you've considered

An alternative could be a debouncer mechanism in-between Sonarr and the client, but this would be cleaner imo.

**Comment by Thirrian** [comment]
https://github.com/Sonarr/Sonarr/issues/4273

**Comment by bakerboy448** [comment]
> [x]  I have searched the existing open and closed issues

Why check the box if no search was performed.

https://github.com/Sonarr/Sonarr/issues?q=sort%3Aupdated-desc+is%3Aissue+season+notification

Duplicate of
- https://github.com/Sonarr/Sonarr/issues/1585
- https://github.com/Sonarr/Sonarr/issues/363
- https://github.com/Sonarr/Sonarr/issues/4273
- https://github.com/Sonarr/Sonarr/issues/1396
- https://github.com/Sonarr/Sonarr/issues/6436
 
and others

On Import Complete should help mitigate this a bit

**Comment by rodrigogonegit** [comment]
@mynameisbogdan If I'm not mistaken that event is triggered when the episode/season pack is grabbed and sent to the downlaod client. I would like the notification to be created only once the file(s) are actually grabbed, hence, I am using the File Import event.

**Comment by markus101** [comment]
Sounds like you found what you're looking for.
